dotCMS/src/integration-test/java/com/dotcms/datagen/SiteDataGen.java

```diff
@@ -14,22 +14,38 @@
     private final long currentTime = System.currentTimeMillis();
 
     private String name = "test" + currentTime + ".dotcms.com";
+    private String aliases = null;
+    private boolean isDefault = false;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'isDefault'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5809706)